src/shared/components/paginationControls/paginationControls.module.scss

```diff
@@ -14,4 +14,8 @@
     height: 18px;
     padding: 0 5px 0 5px;
     display: inline-block;
+}
+
+:global(.legacy) .paginationControls button i {
```

Comment:
Is it possible to add a `TODO` comment inside the code itself as a secondary reminder to ourselves? :)